advocacy_docs/supported-open-source/barman/pg-backup-api/03-tasks.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: 'Operations'
+description: 'Instructions for creating and querying tasks operations on the Postgres Backup API'
+tags:
+    - barman
+    - backup
+    - recovery
+    - postgresql
+    - pg-backup-api
+---
+
+### Prerequisites
+
+ You can ask Barman to perform operations on upstream servers. If you plan to use that feature, the following setup is expected:
+
+ * Postgres backup API (pg-backup-api) must be installed on the same server as Barman. Please check [How to install Postgres backup API](01-installation) if you haven't done it already.
+ * SSH passwordless login must be in place among the different hosts.
+ * Ownership of destination directory or write permissions must be granted to the user Barman is going to use.
+ * For security reasons, the API will listen to on localhost only. You need to use some proxy to forward the traffic. The Apache http server was used during our tests. Please [read our notes about how to define a virtual host](02-securing-pg-backup-api#adding-a-virtualhost-definition-for-postgres-backup-api).
+
+### Available endpoins
+
+ * /servers/NAME/operations/
+ * /servers/NAME/operations/OPERATION_ID
+
+NAME is an available Barman server configuration. Usually in a file like /etc/barman.d/myserver.conf
+
+OPERATION_ID is a short string that represents the ID of an operation created by the API
```

Comment:
Maybe add here something like "You can find out how to fetch operations created by the API below"